src/genetic_testing/requirements.txt

```diff
@@ -0,0 +1 @@
+biopython
```

Comment:
This dependency can go in the `pyproject.toml` file instead

Reply:
I believe I will need to have a separate `requirements.txt` file in the folder where my Streamlit app is because when the app is hosted on Streamlit community cloud, the cloud will check for additional dependencies in the requirements.txt and install those packages so that Streamlit app can work well. 

More info here - https://docs.streamlit.io/streamlit-community-cloud/get-started/deploy-an-app/app-dependencies

Reply:
This is fine for the `app` directory, however, since this directory, `genetic_testing` is your main python package source directory, I agree with @carlosgjs  that this file should be removed and dependency specification moved to `pyproject.toml`.

Reply:
My bad, I moved this file from `genetic_testing` to `app` folder. Let me go ahead and delete this. Sorry for the confusion!